readme.md

```diff
@@ -79,3 +79,9 @@ Embark (Adam G.) set out to fix Germline's -haploid and -homoz-only -w_extend is
 * exit codes follow shell-script convention: 1 for error, 0 for success
 * `germline -version` now reports the version number
 * added tests
+
+#### Version 1.6.0-embark
+
+* Support for using only subsets of samples using ``-old_samples sample_list`` and ``-new_samples sample_list`: Samples for the old list will be compared to the new list
```

Comment:
Why the double ticks here? GH markdown just needs a single tick for code-formatting. Also per the style guide name sample_list in some way that denotes the formatting of the file (.txt maybe?)